spec.md

```diff
@@ -263,9 +271,7 @@ The following example shows a CloudEvent serialized as JSON:
     "source" : "/mycontext",
     "eventID" : "A234-1234-1234",
     "eventTime" : "2018-04-05T17:31:00Z",
-    "extensions" : {
-        "comExampleExtension" : "value"
-    },
+    "comExampleExtension" : "value",
```

Comment:
To avoid people's confusion that only key-value pair format is allowed in the main spec.md and extensions.md, please add the following to this example:
“identityLabels” : {
…
}

Reply:
added some text here to address it